src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -289,8 +292,18 @@ export class PiaIntakeService {
       invitee: {
         createdByGuid: user.idir_user_guid,
       },
+      status: Not(PiaIntakeStatusEnum.COMPLETE),
     });
 
+    // Scenario 5: Return COMPLETE PIAs only if queried and if the user is a drafter, invitee, or MPO
+    if (query.filterByStatus === PiaIntakeStatusEnum.COMPLETE) {
+      whereClause.push({
+        ...commonWhereClause,
+        createdByGuid: user.idir_user_guid,
+        status: PiaIntakeStatusEnum.COMPLETE,
+      });
+    }
+
```

Comment:
still a redundant scenario. We are already adding status filter a few lines below. Find comment`/** filter logic here */`